cfgov/ask_cfpb/models/blocks.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import absolute_import
+
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from django.forms.utils import ErrorList
+
+from wagtail.wagtailcore import blocks
+
+from ask_cfpb.models.django import Answer
+from v1 import blocks as v1_blocks
+from v1.atomic_elements import molecules
+
+
+class AskHeadingLevelBlock(blocks.ChoiceBlock):
+    choices = [
+        ('h2', 'H2'),
+        ('h3', 'H3'),
+        ('h4', 'H4'),
+        ('b', 'bold'),
+    ]
+    classname = 'heading-level-block'
+
+
+class AskHeadingBlock(blocks.StructBlock):
+    text = v1_blocks.HeadingTextBlock(required=False)
+    level = v1_blocks.HeadingLevelBlock(default='h2')
```

Comment:
Is there a reason this is named `level`, but below there's `heading_level` (https://github.com/cfpb/cfgov-refresh/pull/4591/files#diff-a494de7f2d2502b5685c34eb165d5239R119)?